x/auth/client/cli/tx_sign.go

```diff
@@ -166,12 +162,11 @@ func setOutputFile(cmd *cobra.Command) (func(), error) {
 func GetSignCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "sign [file]",
-		Short: "Sign transactions generated offline",
-		Long: `Sign transactions created with the --generate-only flag.
+		Short: "Sign transaction generated offline",
+		Long: `Sign transaction created with the --generate-only flag.
```

Comment:
Disclaimer: I'm not native speaker, so please feel free to correct me if I'm mistaken.

I think you should use either

```suggestion
		Short: "Sign transactions generated offline",
		Long: `Sign transactions created with the --generate-only flag.
```
or

```suggestion
		Short: "Sign a transaction generated offline",
		Long: `Sign a transaction created with the --generate-only flag.
```

Reply:
I think the second one.